docs/code/index.rst

```diff
@@ -13,6 +13,7 @@ the documentation generated from sources linked below.
 .. toctree::
     :maxdepth: 2
 
+    Template Filters <template-filters>
```

Comment:
Would you mind moving it down one line? I'd say that `Class Overview` is something very basic / essential and could be listed as the very first section.

Reply:
Moved in 31f63b84